agent/softwarecontainer-config.in

```diff
@@ -1,27 +1,35 @@
+
+#
+# General SoftwareContainer configuration.
+#
+# Optional values are commented out. All other values are mandatory.
```

Comment:
Can we add that the optional values we have commented out have the default values in this file.